### ISSUE TITLE
Bombs are still selectable after defusing them

### ISSUE THREAD

**Issue by ashbash1987** [ashbash1987/ktanemod-factory#15]
When a bomb is defused and put down, they can be re-selected and will no longer travel correctly on the conveyor belt.

**Comment by CaitSith2** [comment]
All Selectables (more specifically, SelectableArea) are present on gameObject layer 11.  Turns out that if they are moved to any other layer, they can no longer be interacted with until moved back to layer 11.  I figured this out while tracking down why a module in Twitch plays can no longer be interacted with using mouse/motion/touch controls, the moment the camera view has been put onto the module.  Maybe the same thing can be applied to the bomb itself.  Gamepad/gaze is still able to interact with said modules even if the SelectableArea is contained on a different layer than layer 11, as they work by using the child row length system for finding the selectables.
